common/app/com/commercetools/sunrise/myaccount/CustomerSessionHandler.java

```diff
@@ -0,0 +1,65 @@
+package com.commercetools.sunrise.myaccount;
+
+import com.commercetools.sunrise.common.sessions.SessionHandlerBase;
+import com.google.inject.Injector;
+import io.sphere.sdk.customers.Customer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.mvc.Http;
+
+import javax.inject.Inject;
+import java.util.Optional;
+
+import static java.util.Arrays.asList;
+
+public class CustomerSessionHandler extends SessionHandlerBase<Customer> {
+
+    private static final Logger logger = LoggerFactory.getLogger(CustomerSessionHandler.class);
+    private static final String CUSTOMER_ID_SESSION_KEY = "sunrise-customer-id";
```

Comment:
Yes, it could be useful for a quite advanced feature, but I don't really see the need now. Session key conflict could be the only one but I see that very unlikely. Which situation do you have in mind?